From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jean-Baptiste Kempf <jb@videolan.org>
Date: Mon, 29 Jun 2026 17:20:00 +0000
Subject: [PATCH] meson: look for prefixed mingw dlltool

Contrib MinGW builds may only have the target-prefixed dlltool in PATH.
---
 src/meson.build | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index a2f786d..87657f6 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -28,1 +28,4 @@
-  dlltool = find_program('llvm-dlltool', 'dlltool')
+  dlltool = find_program('llvm-dlltool', 'dlltool',
+                         'x86_64-w64-mingw32-dlltool',
+                         'i686-w64-mingw32-dlltool',
+                         'aarch64-w64-mingw32-dlltool')
-- 
2.51.0
